space-plugins/nuxt-base/composables/useAppBridge.ts

```diff
@@ -81,15 +101,6 @@ const useAppBridgeMessages = () => {
 		}
 	};
 
-	onUnmounted(() => {
-		// @ts-ignore not sure how to solve this
-		document.removeEventListener('message', eventListener);
```

Comment:
I changed this to `window.removeEventListener('message', eventListener);` is this ok? Or do you think it will affect something it shouldnt @eunjae-lee

Reply:
oh, you're right. it's window.removeEventListener 🤦🏻‍♂️